checkov/terraform/checks/resource/tencentcloud/VPCSecurityGroupRuleSet.py

```diff
@@ -0,0 +1,30 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_value_check import \
+    BaseResourceCheck
+
+
+class VPCSecurityGroupRuleSet(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure VPC security group rule not accept all traffic"
+        id = "CKV_TC_8"
+        supported_resources = ['tencentcloud_security_group_rule_set']
+        categories = [CheckCategories.ENCRYPTION]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf) -> CheckResult:
+        if conf.get("ingress"):
+            for i in conf["ingress"]:
+                if i.get("action") and i["action"][0] != "ACCEPT":
+                    continue
+                if i.get("cidr_block") and i["cidr_block"][0] != "0.0.0.0/0":
+                    continue
+                if i.get("protocol") and i["protocol"][0] != "ALL":
```

Comment:
Not sure protocol matters. It it's open to all CIDR blocks and ports, any protocol will be bad and removing this would make it more likely to flag bad practices.

Reply:
It’s seem no problem. Only action=="ACCEPT" and cidr_block=="0.0.0.0/0" and protocol=="ALL" and port=="ALL" is failed, others are passed

Reply:
I understand it's technically accurate, but if the user has action=="ACCEPT" and cidr_block=="0.0.0.0/0" and and port=="ALL" and *protocol=="TCP"*, it's still bad. I'm just suggesting it could be a little broader if you skip that part of the check and still be a good check and catch more things. Up to you. We can still merge it this way.